kaspresso/src/main/kotlin/com/kaspersky/kaspresso/docloc/rule/ThemeRule.kt

```diff
@@ -0,0 +1,32 @@
+package com.kaspersky.kaspresso.docloc.rule
+
+import androidx.appcompat.app.AppCompatDelegate
+import com.kaspersky.kaspresso.logger.UiTestLogger
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class ThemeRule internal constructor(
+    private val changeAppTheme: Boolean,
+    private val logger: UiTestLogger
+) : TestRule {
+
+
+    override fun apply(base: Statement, description: Description): Statement {
+        return object : Statement() {
+            override fun evaluate() {
+                logger.i("DocLoc: changeAppThemeRule started")
+                try {
+                    if (changeAppTheme) {
+                        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
+                        base.evaluate()
+                        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
+                        base.evaluate()
+                    }
+                } finally {
+                    AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM)
```

Comment:
Would be better to save preferences before test and restore it after instead of setting "follow system" unconditionally

Reply:
I've changed to use UNSPECIFIED, this is default value and also used to "reset" chosen night mode in app